tsconfig.json

```diff
@@ -18,5 +18,5 @@
   },
   "exclude": ["node_modules"],
   "files": ["./hardhat.config.ts"],
-  "include": ["src/**/*", "tasks/**/*", "test/**/*", "deploy/**/*"]
+  "include": ["src/**/*", "tasks/**/*", "test/**/*", "deploy/**/*", "types/"]
```

Comment:
This is necessary for things like `etyhers.getContractAt("Greeter", ...` to infer the proper types. It's also what typechain recommends (check the suggested tsconfig [here](https://github.com/dethcrypto/TypeChain/tree/master/packages/hardhat)).